src/database/migration/index.ts

```diff
@@ -2,7 +2,7 @@
 import LocalDatabase from '..';
 import allMigrationsFile from './file/index';
 
-export const TARGET_MIGRATION_VERSION = 5;
+export const TARGET_MIGRATION_VERSION = 6;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `TARGET_MIGRATION_VERSION` has been incremented from 5 to 6. Ensure that the corresponding migration file for version 6 exists and is correctly implemented. If not, this could lead to errors during database migration.

```diff
- export const TARGET_MIGRATION_VERSION = 5;
+ export const TARGET_MIGRATION_VERSION = 6;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->